test/components/DayPicker_spec.jsx

```diff
@@ -16,6 +17,8 @@ import {
   VERTICAL_SCROLLABLE,
 } from '../../src/constants';
 
+moment.locale('en');
```

Comment:
this seems like a side effect that shouldn't run at module level - perhaps it should run in a global beforeEach?